devicetypes/smartthings/Orvibo-Contact-Sensor.src/Orvibo-Contact-Sensor.groovy

```diff
@@ -166,10 +166,11 @@ def getBatteryPercentageResult(rawValue) {
 	log.debug "Battery Percentage rawValue = ${rawValue} -> ${rawValue / 2}%"
 	def result = [:]
 	def manufacturer = getDataValue("manufacturer")
+	def application = getDataValue("application")
 	if (0 <= rawValue && rawValue <= 200) {
 		result.name = 'battery'
 		result.translatable = true
-	if (manufacturer == "Third Reality, Inc" || manufacturer == "THIRDREALITY") {
+	if ((manufacturer == "Third Reality, Inc" || manufacturer == "THIRDREALITY") && application <= "17") {
```

Comment:
you're comparing a string with `<=`

please make a new PR that reverts this and also fix it in all of your other PRs.